scripts/src/main/resources/scripts/command/migration

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+for migration in "$(dirname "$0")/../migrations"/*
+do
+  from_version="${1}"
+  to_version="${migration/*\//}"
+  doVersionCompare "${from_version}" "${to_version}"
+  result="${?}"
+  if [ "${result}" -eq 0 ] || [ "${result}" -eq 2 ]
```

Comment:
The filesystem can give us files (migrations) in random order.
Is the an option for `sort` so we can compare such that the segment numbers are sorted numeric and not alphabetical (e.g. `1`, `2`, ..., `10` and not `1`, `10`, ... - also `1`, `02`, `003`). Otherwise we would need to implement sorting ourselves using `doVersionCompare` what will be very slow...

Reply:
I have added a `sort -n` command to order the migrations script numerically.

Check commit b548765c7441a3edba09d716db9d91f3c307bbaf

Reply:
Seems to be the perfect solution (simple and efficient). As long as we do not change our versioning scheme to something odd it should work once and forever. Thanks.